script/options.js

```diff
@@ -40,9 +40,14 @@ function gatherFromCommandLine() {
 }
 
 function applyDeprecatedBuildtypes(options) {
-  const deprecatedEnvironments = [environments.DEVELOPMENT];
-
-  const isDeprecated = deprecatedEnvironments.includes(options.buildtype);
+  // All Vets.gov environments generate redirects to VA.gov
+  const deprecatedEnvironments = new Set([
+    environments.DEVELOPMENT,
+    environments.STAGING,
+    environments.PRODUCTION,
+  ]);
+
+  const isDeprecated = deprecatedEnvironments.has(options.buildtype);
```

Comment:
I merged this PR, https://github.com/department-of-veterans-affairs/vets-website/pull/9043, into this branch so that we can clearly see that Vets.gov is building as redirects, and nothing flies under the radar.

Reply:
Ugh, I'm going to revert this out of fear that it'll be accidentally merged.